### ISSUE TITLE
chart does not show the x-axis value

### ISSUE THREAD

**Issue by kikikiblue** [ankane/chartkick#353]
Hi. Thanks for this great gem!!
I am making a chart but can't show the x-axis value. 
rails code is like this.
`<%= bar_chart User.group(:provider).group_by_week(:created_at, last: 8).count %>`
it shows this chart.
<img width="556" alt="2017-04-19 6 12 50" src="https://cloud.githubusercontent.com/assets/481619/25172426/f0a5f472-252b-11e7-82d2-c7d70c454b9b.png">

generated htmls.
`<div id="chart-1" style="height: 300px; width: 100%; text-align: center; color: #999; line-height: 300px; font-size: 14px; font-family: 'Lucida Grande', 'Lucida Sans Unicode', Verdana, Arial, Helvetica, sans-serif;"><iframe class="chartjs-hidden-iframe" tabindex="-1" style="display: block; overflow: hidden; border: 0px; margin: 0px; top: 0px; left: 0px; bottom: 0px; right: 0px; height: 100%; width: 100%; position: absolute; pointer-events: none; z-index: -1;"></iframe><canvas height="600" width="1072" style="display: block; height: 300px; width: 536px;"></canvas></div>`

generated javascripts
```
<script type="text/javascript">
  new Chartkick.BarChart("chart-1", [{"name":"facebook","data":[["2017-02-26",6],["2017-03-05",9],["2017-03-12",7],["2017-03-19",7],["2017-03-26",229],["2017-04-02",3],["2017-04-09",0],["2017-04-16",0]]},{"name":"google","data":[["2017-02-26",0],["2017-03-05",0],["2017-03-12",0],["2017-03-19",0],["2017-03-26",6],["2017-04-02",2],["2017-04-09",0],["2017-04-16",0]]},{"name":"email","data":[["2017-02-26",1],["2017-03-05",1],["2017-03-12",1],["2017-03-19",4],["2017-03-26",23],["2017-04-02",0],["2017-04-09",0],["2017-04-16",1]]}], {});
</script>
```

please let me know how can I fix this problem.

**Comment by ankane** [comment]
Hey @kikikiblue, thanks for the detailed report. Unfortunately, I'm unable to reproduce.

![screen shot 2017-04-19 at 8 41 09 pm](https://cloud.githubusercontent.com/assets/220358/25212681/a67d20d6-2540-11e7-8ca4-0392c9a670b8.png)

**Comment by kikikiblue** [comment]
I solved it. 
I made this graph by requesting html in ajax and shipped it on bootstrap modal. 
(chartkick javascript is alreay loaded inside application.js by the parent window)
I called below js and it start to show the x-axis value.
Your comment was good hint for me to solve this. 
Thank you for your quck comment and thank you again for great gem!!

```
$('#ajaxmodalwindow').on('shown.bs.modal', function (e) {
  Chartkick.eachChart( function(chart) {
    chart.redraw();
  });
});
```
